suites/reef/rgw/tier-1_rgw_multisite.yaml

```diff
@@ -0,0 +1,286 @@
+# Test suite for evaluating RGW multi-site deployment scenario.
```

Comment:
This suite and, suites/reef/rgw/sanity_rgw_multisite.yaml are identical,
May need to  add few basic multi-site test-cases here

Reply:
yes its intentional, we will be adding tc in the future. And this is to reduce occurance of dublicate tc.
Testsuites with prefix "sanity" can be used for hotfix testing and we will ebe adding basic tc of each feature in it